schema/lc.proto

```diff
@@ -172,6 +172,41 @@ message EntryList {
     repeated Entry entries = 1;
 }
 
+message LabelsErrorResponse {
```

Comment:
IIRC we're passing errors thorough standard GRPC error handling, @qrdl @mmeloni can you confirm?

Reply:
Yes, this is correct, usually on error we just return non-nil `error`

Reply:
fixed